### PR TITLE
Feature/sound volume sliders

### PR DESCRIPTION
**Pull request by JucanAndreiDaniel**
# To test
1. Test volume in main menu
2. Load world and test if volume is properly set
3. Restart the game and check if sound volume is properly set

# Testing
* [x] Merged _main_ into this branch and tested with the latest features
* [x] Tested with PCVR
* [x] Tested with Pico / Quest

### CODE REVIEW

File: Assets/GothicVR/Scenes/Bootstrap.unity [hunk 1]
```diff
@@ -1258,6 +1259,102 @@ Transform:
   m_Children: []
   m_Father: {fileID: 1103036566}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!82 &1283201087
+AudioSource:
```
Comment (JaXt0r) [inline Assets/GothicVR/Scenes/Bootstrap.unity:1263]: Why do we need an AudioSource on Bootstrap scene? Wouldn't it be better suited inside Menu.unity or General.unity?

File: Assets/GothicVR/Scripts/UI/AudioMixerHandler.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        void Awake()
+        {
+            float oldVolume = PlayerPrefs.GetFloat(volumePlayerPrefName, 1f);
+            audioVolumeSlider.value = oldVolume;
+        }
+
+        public void SliderUpdate(float value)
+        {
+            PlayerPrefs.SetFloat(volumePlayerPrefName, value);
+            audioMixer.audioMixer.SetFloat(volumePlayerPrefName, Mathf.Log10(value) * 20);
```
Comment (JaXt0r) [inline Assets/GothicVR/Scripts/UI/AudioMixerHandler.cs:22]: Why log(value) * 20?
Can a slider value like 0-100 be sufficient?

**Review by JaXt0r — CHANGES_REQUESTED** [review]

File: Assets/GothicVR/Scenes/Bootstrap.unity (hunk 1 above)
Comment (JucanAndreiDaniel) [reply Assets/GothicVR/Scenes/Bootstrap.unity:1263]: Before we already had the BackgroundMusic gameobject being available in the Bootstrap scene, but the audio source was created in script in MusicManager.
I just skipped the step in the script to create the Audio source, just so i can already set the output mixer, letting me control the volume perceived by the audio listener without directly controlling the volume of the audio source

File: Assets/GothicVR/Scripts/UI/AudioMixerHandler.cs (hunk 2 above)
Comment (JucanAndreiDaniel) [reply Assets/GothicVR/Scripts/UI/AudioMixerHandler.cs:24]: Volume and loudness are not the same, volume can be linear but loudness is logarithmic
https://www.msdmanuals.com/home/multimedia/table/measurement-of-loudness

**Review by JaXt0r — COMMENTED** [review]
Thanks for clarifications. Code looks good now. Will test the implementation soon.

**Pull request merged**